src/Symfony/Component/DependencyInjection/Exception/ContextElementNotFoundException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Exception;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ContextElementNotFoundException extends InvalidArgumentException
+{
+    public function __construct($key, \Exception $previous = null)
```

Comment:
This exception is only thrown by the `Context` class and this class is final. I don't see the need of this ?